python-controller/inkkeys/protocol.py

```diff
@@ -19,7 +19,7 @@ class CommandCode(Enum):
     INFO = "I"
 
 class RefreshTypeCode(Enum):
-    PARTIAL = "p"
+    PARTIAL = "f"
```

Comment:
This was the line that made the screen to work. For some reazon the partial refresh is not working with the v2 board, so this can be a temporary fix until we find a proper soltuion with a library update or some firmware refactor

Reply:
Nice. Let's remove the PARTIAL completely then. Is just used [here](https://github.com/gkaffka/inkkeys/blob/main/python-controller/inkkeys/device.py#L163), this line could be `self.sendToDevice(CommandCode.REFRESH.value + " " + RefreshTypeCode.FULL.value)`.

I bet this can decrease the screen lifetime or sth. But we can live with it I think. 😅

Reply:
👍 Nice! I will do it, but before removing it completely I just wanna check the screen and the lib docs, maybe there is a small change we can do to make the partial refresh work 🤞 At least now we know what to look for 👯

Reply:
Reverted this change because there is now a fix that enable for partial refreshing :) 
https://www.reddit.com/r/thereoughtabe/comments/s9sgox/inkkey_display_wont_update/htz2eg6